lib/chef/provider/package/homebrew.rb

```diff
@@ -115,19 +158,33 @@ def current_installed_version
         # forward project.
         #
         # https://github.com/Homebrew/homebrew/wiki/Acceptable-Formulae#stable-versions
-        def candidate_version
-          brew_info["versions"]["stable"]
-        end
+        #
+        # @param [String] package name
+        #
+        # @returns [String] package version
+        def available_version(i)
+          p_data = package_info(i)
 
-        private
+          # nil means we couldn't find anything
+          return nil if p_data.nil?
 
-        def get_response_from_command(*command)
+          p_data["versions"]["stable"]
```

Comment:
and actually down here p_data could be `{}` which would trainwreck this method.

Reply:
That's handled now